README.md

```diff
@@ -49,6 +49,31 @@ create_table :emails do |t|
  end
 ```
 
+You can choose whether or not users can login with an email address that is not the primary email address.
+
+```ruby
+Devise::MultiEmail.configure do |config|
+  # Default is `false`
+  config.only_login_with_primary_email = true
+end
+```
+
+The `autosave` is automatically enabled on the `emails` association by default. This is to ensure the `primary`
+flag is persisted for all emails when the primary email is changed. When `autosave` is not enabled on the association,
+only new emails are saved when the parent (e.g. `User`) record is saved. (Updates to already-persisted email records
+are not saved.)
+
+If you don't want `autosave` to be enabled automatically, you can disable this feature. What this will do is
+enable alternative behavior, which adds an `after_save` callback to the parent record, to update the `primary`
+flag for emails using an `update_all` SQL UPDATE statement.
```

Comment:
@joelvh Should this be updated since `updated_all` is not used ?   other parts LGTM, our app works fine.